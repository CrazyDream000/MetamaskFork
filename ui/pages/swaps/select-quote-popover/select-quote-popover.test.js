import React from 'react';

import { renderWithProvider } from '../../../../test/jest';
import SelectQuotePopover from '.';

const createProps = (customProps = {}) => {
  return {
    onClose: jest.fn(),
    onSubmit: jest.fn(),
    swapToSymbol: 'ETH',
    initialAggId: 'initialAggId',
    onQuoteDetailsIsOpened: jest.fn(),
    ...customProps,
  };
};

describe('SelectQuotePopover', () => {
  it('renders the component with initial props', () => {
    const { container } = renderWithProvider(
      <SelectQuotePopover {...createProps()} />,
    );
    expect(container).toMatchSnapshot();
  });
});
