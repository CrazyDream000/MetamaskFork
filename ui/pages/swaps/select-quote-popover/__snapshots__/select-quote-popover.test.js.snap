// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SelectQuotePopover renders the component with initial props 1`] = `
<div>
  <div
    class="select-quote-popover"
  />
</div>
`;
